snappy.cc

```diff
@@ -721,8 +784,8 @@ static inline uint32 ExtractLowBytes(uint32 v, int n) {
   return _bzhi_u32(v, 8 * n);
 #else
   // This needs to be wider than uint32 otherwise `mask << 32` will be
-  // undefined.
-  uint64 mask = 0xffffffff;
+  // undefined. 
+  uint64 mask = ~0;
```

Comment:
(commenting the 2nd time here, comment disappeared)

are you absolutely(!) sure to change "0xffffffff" to  "~0" here? This looks like modifying the existent code, which introduces additional review effort. If confident, provide a comment (possibly with a link to relevant docu).